es/03_3_Configurando_Su_Billetera.md

```diff
@@ -0,0 +1,136 @@
+```
+El archivo `mywallet.txt` en su directorio personal tendrá una larga lista de claves privadas, direcciones y otra información. Eso sí, ¡nunca querrá poner estos datos en un archivo de texto sin formato en una configuración de Bitcoin con fondos reales!
+
+Luego puede recuperarlo con `bitcoin-cli importwallet`.
+```
+$ bitcoin-cli importwallet ~/mywallet.txt
+```
+¡Pero tenga en cuenta que esto requiere un nodo sin podar!
+```
+$ bitcoin-cli importwallet ~/mywallet.txt
+error code: -4
+error message:
+Importing wallets is disabled when blocks are pruned
+```
+
+## Opcional: vea sus claves privadas
+
+A veces, es posible que desee ver las claves privadas asociadas con sus direcciones de Bitcoin. Quizás quiera poder firmar un mensaje o gastar bitcoins desde una máquina diferente. Quizás solo desee hacer una copia de seguridad de ciertas claves privadas importantes. También puede hacer esto con su archivo de volcado, ya que es legible por humanos.
+```
+$ bitcoin-cli dumpwallet ~/mywallet.txt
+{
+  "filename": "/home/standup/mywallet.txt"
+}
+```
+Lo más probable es que solo desee ver la clave privada asociada con una dirección específica. Esto se puede hacer con el comando `bitcoin-cli dumpprivkey`.
+```
+$ bitcoin-cli dumpprivkey "moKVV6XEhfrBCE3QCYq6ppT7AaMF8KsZ1B"
+cTv75T4B3NsG92tdSxSfzhuaGrzrmc1rJjLKscoQZXqNRs5tpYhH
+```
+Luego, puede guardar esa clave en un lugar seguro, preferiblemente en un lugar que no esté conectado a Internet.
+
+También puede importar cualquier clave privada, desde un volcado de billetera o un volcado de clave individual, de la siguiente manera:
+```
+$ bitcoin-cli importprivkey cW4s4MdW7BkUmqiKgYzSJdmvnzq8QDrf6gszPMC7eLmfcdoRHtHh
+```
+Nuevamente, espere que esto requiera un nodo sin podar. Espere que esto tome un tiempo, ya que `bitcoind` debe volver a leer todas las transacciones pasadas, para ver si hay algunas nuevas a las que debería prestar atención.
+
+> :information_source: **NOTA:** Muchas carteras modernas prefieren [códigos mnemotécnicos](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) para generar las semillas necesarias para crear las claves privadas. Esta metodología no se utiliza en `bitcoin-cli`, por lo que no podrá generar listas de palabras útiles para recordar sus claves privadas.
+
+_Ha estado escribiendo esa dirección Bitcoin que generó muchas veces, mientras estaba firmando mensajes y ahora exportando llaves. Si cree que es una molestia, estamos de acuerdo. También es propenso a errores, un tema que abordaremos en la siguiente sección._
+
+## Resumen: configuración de su billetera
+
+Necesita crear una dirección para recibir fondos. Su dirección se almacena en una billetera, de la que puede hacer una copia de seguridad. También puede hacer mucho más con una dirección, como descargar su clave privada o usarla para firmar mensajes. Pero realmente, crear esa dirección es _todo_ lo que necesita hacer para recibir fondos de Bitcoin.
+
+## Que sigue?
```

Comment:
`## ¿Que sigue?`

Reply:
Is there a reason to no apply this change?

Reply:
yes, basically i missed it :-)